pkg/multi-tenancy/read/read.go

```diff
@@ -0,0 +1,39 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license.
+
+package read
+
+import (
+	"fmt"
+
+	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/timescale/promscale/pkg/multi-tenancy/config"
+)
+
+type readAuthorizer struct {
+	authorizerConfig
```

Comment:
not sure I understand the difference between a readAuthorizer and an authorizeConfig - why not just make this one struct